packages/encoder/test/wrap-fc.test.ts

```diff
@@ -313,6 +311,19 @@ const BoolInput = (value: boolean) => {
   return fc.oneof(...addBoxes(representations).map(fc.constant));
 };
 
+function testDecimalAgreementWithTolerance(actual: Big, expected: Big): void {
+  const negativeLog10OfRelativeTolerance = (2 ** 53).toString().length - 1; //15
```

Comment:
What is the significance of this number?

Reply:
As mentioned above, it's the precision of a `number` expressed in decimal places (approximately) as opposed to bits.

Reply:
I guess really, instead of using `2 ** 53`, I should use the constant `Number.MAX_SAFE_INTEGER`.  I guess I'll go change when I have time.  (Also, arguably, instead of `num.toString().length - 1`, I should use `Math.floor(Math.log10(num))`... I might change that as well.)

Reply:
OK, I've replaced this with an expression that is hopefully clearer.